account_credit_control/README.rst

```diff
@@ -16,14 +21,24 @@ You are able to specify a particular policy for one partner or one invoice.
 Usage
 -----
 
-Menu entries are located in ``Accounting > Periodical Processing > Credit
-Control``.
+Menu entries are located in ``Accounting > Report > Credit Control``.
 
 Create a new "run" in the ``Credit Control Run`` menu with the controlling date.
-Then, use the ``Compute credit lines`` button. All the credit control lines will
+Then, use the ``Compute Credit Lines`` button. All the credit control lines will
 be generated. You can find them in the ``Credit Control Lines`` menu.
 
 On each generated line, you have many choices:
  * Send a email
  * Print a letter
  * Change the state (so you can ignore or reopen lines)
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Nicolas Bessi (Camptocamp)
+* Guewen Baconnier (Camptocamp)
+* Sylvain Van Hoof (Okia SPRL) <sylvain@okia.be>
```

Comment:
You must add Try button:
`
.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
   :alt: Try me on Runbot
   :target: https://runbot.odoo-community.org/runbot/92/10.0`